src/main/java/io/kestra/plugin/surrealdb/SurrealDBConnectionInterface.java

```diff
@@ -0,0 +1,58 @@
+package io.kestra.plugin.surrealdb;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+
+public interface SurrealDBConnectionInterface {
+
+	@Schema(
+			title = "Use tls to connect"
```

Comment:
```suggestion
			title = "Whether to TLS to connect"
```

Abbreviations must be uppercase and boolean are usually describe with `Whether`